vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -21,15 +21,11 @@ packages:
      - libpolkit-agent-1-0
      - libpolkit-backend-1-0
      - libpolkit-gobject-1-0
-    pip: # ansible uses these packages in python2 to run on the local machine
-     - libvirt-python==5.10.0
-     - lxml
-     - six
     pip3:
      - virtualbmc
      - lxml
      - netaddr
-     - libvirt-python
+     - libvirt-python==5.10.0
```

Comment:
Is it worth adding a comment saying why this has to be pinned?

Reply:
When latest release of `libvirt-python` came out integration CI started to fail. Then, we decided to switch to `v5.10.0`  to avoid CI failure.  
I thought that we need the same in case of python3 as well, but apparently after my test, the latest versionof  `libvirt-python` works fine in dev-env.

Reply:
So, switching back to the latest